scripts/upgrade

```diff
@@ -72,14 +71,6 @@ ynh_app_setting_delete --app=$app --key=public_stats
 ynh_app_setting_delete --app=$app --key=private
 ynh_app_setting_delete --app=$app --key=local_timeline
 
-# Close a port
-
-if yunohost firewall list | grep -q "\- $port$"
-then
-	echo "Close port $port" >&2
-	yunohost firewall disallow TCP $port 2>&1
-fi
-
```

Comment:
Those lines need to be kept because old version of the package opened the port ...